media-sound/audacity/audacity-3.4.2.ebuild

```diff
@@ -70,6 +70,7 @@ RDEPEND="dev-db/sqlite:3
 	media-libs/libpng:=
 	media-libs/libsndfile
 	media-libs/libsoundtouch:=
+	media-libs/opusfile
```

Comment:
_Thank you for the effort, @vowstar!_

The `opus` USE flag turned out to be quite broken, so unfortunately this isn't a proper fix.
I've submitted a corresponding PR:
  - #35903 

I believe it's better to close the current PR.

Reply:
OK, thanks